.drone.yml

```diff
@@ -0,0 +1,43 @@
+---
+kind: pipeline
+name: amd64
+
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+  - name: validate
```

Comment:
Once `validate` is already included at `test` script the `validate` step wouldn't be required any more, is it??